CompileIO/compile.io
# closes #125

# Addressed Issue(s)

#125

# Goals

Create Mock Poster to build off of for our presentation on 2/13/19.

# Special Considerations

This is not the final product.

# Acceptance Criteria

- The Mock Poster should be in the `Documentation` folder.

# Testing

- View the Mock Poster. Make sure it looks okay (no typos, other mistakes, etc.)
- Make sure it's in the `Documentation` folder

# Branch

`poster-draft-125`
